docusaurus.config.js

```diff
@@ -425,12 +341,14 @@ const config = {
             label: "Wallet",
           },
           {
-            to: "/wallet/how-to/connect/set-up-sdk",
+            to: "/wallet/how-to/use-sdk",
             label: "SDK",
+            className: "navbar-item-beta",
           },
           {
             to: "snaps",
             label: "Snaps",
+            className: "navbar-item-beta",
```

Comment:
I'm not sure about this change. What is the reasoning?

Reply:
It appears from the [Snaps homepage](https://metamask.io/snaps/) that Snaps is in "Open Beta." From separate discussions with @ezgi-cengiz and @vandan, we considered if we were to add the "Beta" tag to the SDK, it might make sense to add it to Snaps for consistency and transparency.

I'd definitely like to hear the Snaps team perspective on the release stage and possible solutions for signifying it, considering we have to add caveats to the SDK as well.

Reply:
Yeah, it's "Open Beta" for users. That doesn't mean it's a beta for developers. I know it can be confusing to have different messaging for the two different groups, but I think it's worse to imply that Snaps are not production ready when anyone can install a Snap into the MetaMask extension. 

I will discuss with the other product leads and get back to you.

Reply:
OK please remove the BETA tag on both. We will not use the caveat here in the header. 

I will talk to the SDK team about which SDK packages need the caveat in the sidebar and will circle back on whether any wallet APIs need the same. In the future we will caveat such things the same way we have a Flask caveat in the Snaps docs.